### ISSUE TITLE
[REQ] fix license on GH

### ISSUE THREAD

**Issue by forart** [xris1658/musec#7]
Hi there, 1st of all thanks for your awesome (voluntary) work !

Since we've doxed this git in our [HyMPS project](https://github.com/forart/HyMPS) (under the [DAW section](https://github.com/forart/HyMPS/blob/main/DAWs.md)), can you please fix licensing terms for it ?

Our "problem" is that does not generates a correct badge:

![](https://flat.badgen.net/github/license/xris1658/musec/?label=LICENSE)
(generate url: https://flat.badgen.net/github/license/xris1658/musec/?label=LICENSE)

This could easily be fixed by choosing a "correct" license file through the GH's wizard.

Although it may sounds like a minor aspect, this could help other devs to understand how reuse your code in other open projects and vice-versa.

Last but not least, let us know how (in your opinion) we could improve our categorizations and links to resources in order to favor collaboration (and therefore evolution) between developers of listed projects.

Thanks in advance.

**Comment by xris1658** [comment]
Thanks for your support!

I've already found the license issue, but I'm just not pretty sure which license I should use. This project uses VST2 and ASIO, both of which requires to sign a license agreement included in their SDKs (which I haven't signed yet, so I can't distribute the build according to the agreement). And it seems like the license agreement is not compatible with several open source licenses (e.g. GPLv3 used by [Audacity](https://manual.audacityteam.org/man/asio_audio_interface.html)).

Sorry for my bad English as I'm not a native speaker.

**Comment by forart** [comment]
Well, many open projects that uses closed 3rd-party components choosed MIT or L-GPL but - as you can easily get from [our page](https://github.com/forart/HyMPS/blob/main/DAWs.md#subsections) - seems that the majority of DAWs (which should provide functionalities similar to yours) have opted for GPL v3 instead.

As said, choosing the "right" license is extremely important 'cause it could determine the success and longevity of your project.

Hope that helps.

**Comment by forart** [comment]
While waiting for your choice, we want to inform you that we've recently introduced a "dev-oriented" grouping+sorting, so projects are now listed by the following sorting chain:

Section > Subsection > Language > License > Alphabetical

Let us know if your git is rightly placed ([here](https://github.com/forart/HyMPS/blob/main/DAWs.md#generic-)) or you need a different one.

Do you think this new approach could foster collaboration between projects better? Any observation/suggestions?

Thanks again and, if you like/need, let us know (by participating in [our discussions](https://github.com/forart/HyMPS/discussions) too) how can we better support listed projects.

Last but not least, Happy New Year !

**Comment by xris1658** [comment]
Quick update: I removed support of VST2 plugins.

From October 2018, Steinberg stopped licensing developers who want to make something using VST2. Though still widely used by end users, it's safe to say VST2 is an inferior API than other plugin/host APIs that came to life later, like VST3 and CLAP. So I decided to remove VST2 support.

For now the only licensing issue comes from ASIO support. I still have to read the license agreement, or just check other open-source projects that use ASIO, before choosing the proper open-source license.
